.gitignore

```diff
@@ -122,4 +122,19 @@ env/.env.local
 env/.env.staging
 .env*
 /dotnet/samples/NuGet.Config
-/dotnet/samples/LocalPkg/
\ No newline at end of file
+/dotnet/samples/LocalPkg/
+/dotnet/samples/08.datasource.azureaisearch/bin/Debug/net6.0
+/dotnet/samples/08.datasource.azureaisearch/obj
+/dotnet/samples/08.datasource.azureaisearch/.vs
+/dotnet/samples/08.datasource.azureaisearch/AzureAISearchBot/bin/Debug
+/dotnet/samples/08.datasource.azureaisearch/AzureAISearchBot/obj
+/dotnet/samples/08.datasource.azureaisearch/AzureAISearchBot/AzureAISearchBot.csproj.user
```

Comment:
Do these really need to go in the root .gitignore?

Reply:
fixed